extensions/wikia/SpitfiresContributionExperiments/templates/MyProfileTourExperimentStep3.mustache

```diff
@@ -0,0 +1,8 @@
+<div class="my-profile-step step-3"></div>
+<a class="my-profile-header-go-back">BACK</a>
+<h3 class="my-profile-welcome-header">Your profile is almost ready!</h3>
+<form class="my-profile-form">
+	<label class="my-profile-label" for="my-profile-gaming-platforms">Tell us a few words about yourself:</label>
+	<textarea class="my-profile-textarea" id="my-profile-gaming-platforms">{{answer}}</textarea>
```

Comment:
Those templates are loaded interchangeably. Besides you proposal is obviously a good idea but given the time and experimental nature and throw away code here I think we can treat this optional.